File: leetcode/1450.在既定时间做作业的学生人数.py
#
# @lc app=leetcode.cn id=1450 lang=python3
#
# [1450] 在既定时间做作业的学生人数
#
from typing import List


# @lc code=start
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        return sum(
            1 for i in range(len(startTime))
            if startTime[i] <= queryTime <= endTime[i]
        )

# @lc code=end

File: leetcode/1450.在既定时间做作业的学生人数.cpp
#include <istream>
#include <vector>
using namespace std;
/*
 * @lc app=leetcode.cn id=1450 lang=cpp
 *
 * [1450] 在既定时间做作业的学生人数
 */

// @lc code=start
class Solution
{
public:
    int busyStudent(vector<int> &startTime, vector<int> &endTime, int queryTime)
    {
        int res = 0;
        for (size_t i = 0; i < startTime.size(); i++)
        {
            if (startTime[i] <= queryTime && queryTime <= endTime[i])
                res++;
        }
        return res;
    }
};
// @lc code=end
